bundles/automation/org.eclipse.smarthome.automation.rest/src/main/java/org/eclipse/smarthome/automation/rest/internal/RuleResource.java

```diff
@@ -1,4 +1,8 @@
 /**
+ * Copyright (c) 2017 by Bosch Software Innovations GmbH. All rights reserved,
```

Comment:
Unfortunately it is same problem as here: https://github.com/eclipse/smarthome/pull/3671/files/025f8b55b62675de1c46753f0c0fccc32e7697b8#r129799279